src/TestProblem/Hydro/CCSN/Flag_CCSN.cpp

```diff
@@ -63,10 +63,7 @@ bool Flag_CoreCollapse( const int i, const int j, const int k, const int lv, con
 
 
 // (2) always refined to highest level in the region with r < 30 km
-   if (  !MaxRefine  &&  ( r < amr->dh[lv] )  &&  ( amr->dh[lv] * sqrt(0.75) * UNIT_L >= 3e6 )  )
-      Flag = true;
-
-   if (  !MaxRefine  &&  ( r * UNIT_L < 3e6 )  )
+   if (   !MaxRefine  &&  (  ( r * UNIT_L < 3e6 ) || ( r < amr->dh[lv] )  )   )
```

Comment:
It is fine here. However, I prefer the following format to make it more readable 

~~~
if ( !MaxRefine )
{
// description for this additional condition
   if ( r < amr->dh[lv] )
      ...

   if ( r * UNIT_L < 3e6 ) 
      ...
}
~~~

BTW, have you checked whether removing these conditions will not ruin anything?

Reply:
I have changed the format. It is being tested. I will let you know once it is done.